lib/npg/samplesheet/novaseq_xseries.pm

```diff
@@ -158,9 +158,60 @@ sub _build_batch_id {
   return $batch_id;
 }
 
+
+=head2 index_read_length
+
+An array containing the length of the first and the second (if applicable)
+indexing read.
+
+If not set, is computed as the longest first and second barcode as reported
+by the LIMS system.
+
+=cut
+
+has 'index_read_length' => (
```

Comment:
Is this meant to be used as an override? Should we care if only one is provided at executing time but there are two lengths in LIMS?

Reply:
This is when we want to do something semi-manually. Prior to the start of the run we only have LIMS information. If we want to run DRAGEN off-board, then we will find out what happenned from RunInfo.xml As we know only two well, all sorts of discrepancies are possible.